packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -181,6 +181,13 @@ class PlatformPolygon {
   final int zIndex;
 }
 
+/// Join types for polyline joints. The indices of this enum must match those used internally by the maps SDK.
```

Comment:
These comments should be updated once my comment about this being problematic is addressed.